packages/venia-ui/lib/components/CartPage/ProductListing/product.css

```diff
@@ -1,8 +1,16 @@
 .root {
     display: grid;
     grid-gap: 0.5rem 1rem;
-    grid-template-areas: 'image details kebab';
-    grid-template-columns: 100px 1fr min-content;
+    grid-template-areas: 'item quantity kebab';
```

Comment:
This change broke the responsive flow of quantity below options, will need to be resolved.

<img width="464" alt="Screen Shot 2020-07-20 at 8 48 15 AM" src="https://user-images.githubusercontent.com/462953/87945122-eb839c00-ca65-11ea-86a0-f310e927889e.png">

Reply:
My bad. Fixed it in bf8f2c3.

![image](https://user-images.githubusercontent.com/35203638/87974051-10d8d000-ca8f-11ea-90f4-f2582808c1a3.png)